s2d-oms-mgmt-solution/azuredeploy.parameters.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "omsLogAnalyticsWorkspaceName": {
+            "value": "changeme"
+        },
+        "omsLogAnalyticsWorkspaceLocation": {
+            "value": "East US"
+        },
+        "omsLogAnalyticsWorkspaceApiVersion":{
```

Comment:
I will change this but I am trying to write my templates in more modular way like any dev language. I think ARM and this repository should be able to support these methods.